x-pack/plugin/ccr/qa/multi-cluster/build.gradle

```diff
@@ -9,51 +9,53 @@ dependencies {
   testImplementation project(':x-pack:plugin:ccr:qa')
 }
 
-task "leader-cluster"(type: RestIntegTestTask) {
-  mustRunAfter("precommit")
-  systemProperty 'tests.target_cluster', 'leader'
+testClusters {
+  'leader-cluster' {
+    testDistribution = 'DEFAULT'
+    setting 'xpack.license.self_generated.type', 'trial'
+  }
+  'middle-cluster' {
+    testDistribution = 'DEFAULT'
+    setting 'xpack.license.self_generated.type', 'trial'
+    setting 'cluster.remote.leader_cluster.seeds',
+            { "\"${testClusters.named('leader-cluster').get().getAllTransportPortURI().join(",")}\"" }
+  }
 }
 
-testClusters."leader-cluster" {
-  testDistribution = 'DEFAULT'
-  setting 'xpack.license.self_generated.type', 'trial'
+tasks.register("leader-cluster", RestIntegTestTask) {
+  mustRunAfter("precommit")
```

Comment:
Does this not require a `useCluster`?

Reply:
The name matching cluster is always wired here: https://github.com/elastic/elasticsearch/blob/0911d04467ef3f507321a3b92ec831478a5b66cc/buildSrc/src/main/java/org/elasticsearch/gradle/test/RestTestBasePlugin.java#L47